Configuration/Services.yaml

```diff
@@ -27,6 +27,13 @@ services:
               method: handleEvent
               event: TYPO3\CMS\Filelist\Event\ProcessFileListActionsEvent
 
+    DMK\MkContentAi\Backend\EventListener\NewsRecordEventListener:
+        public: true
```

Comment:
This is superfluous, event listeners are public by default (see https://github.com/TYPO3/typo3/blob/v12.4.24/typo3/sysext/core/Classes/DependencyInjection/ListenerProviderPass.php#L81).